README.md

```diff
@@ -32,6 +32,18 @@ The generator is tested via its output when run against `openapi.yml`, viz., the
 
 Beyond what is necessary to test the generator, the Cocina model classes are not tested, i.e., they are assumed to be as specified in `openapi.yml`.
 
+## Releasing
+
+The release process is much like any other gem. First bump the version in `lib/cocina/models/version.rb`, and commit the result. Then run:
+```
+bundle exec rake release
+```
+which pushes the gem to rubygems.org.  Next write up the release notes: https://github.com/sul-dlss/cocina-models/releases .
+
+Finally, you must release versions of [sdr-api](https://github.com/sul-dlss/sdr-api) and [dor-services-client](https://github.com/sul-dlss/dor-services-client/) pinned to this version because [Argo](https://github.com/sul-dlss/argo) depends on both of those. When [dor-services-app](https://github.com/sul-dlss/dor-services-app) is update to use the new models, the clients must be updated at the same time or there is risk of models produced by dor-services-app not being acceptable to the clients.
```

Comment:
sdr-api client and all consumers of client (viz, Google Books)?

Reply:
I was omitting "all consumers of client" because the auto-update script is handling that.  The autoupdate script breaks in the case when we don't update sdr-client and dor-services-client both at the same time and I offered one example how.

Reply:
Doesn't sdr-client need to be updated and released, as it is pinned to a particular cocina-models version: https://github.com/sul-dlss/dor-services-client/blob/master/dor-services-client.gemspec#L26?

Reply:
yes, I typoed sdr-client as sdr-api here.